### PR TITLE
Navbar Component

### PR DESCRIPTION
**Pull request by mehradadimi**
This is responsive.

This PR adds colors to the Theme file, but can all be changed.

The colors are just based on what made sense, did not spend much time to make it look nicer. Your color suggestions are appreciated. 

Let me know if any change is required.

https://github.com/SENG480a-NutriDine/webapp/assets/83256092/fa1a6166-87d7-4e44-8aba-c7af754150c9

### CODE REVIEW

File: nutridine/src/components/navbar/MenuItem.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { Box, Text, useColorModeValue } from "@chakra-ui/react";
+
+export const MenuItem = ({ children, onClick, isActive, ...rest }) => {
+  const activeBg = useColorModeValue("light.primary.500", "dark.primary.600");
+  const paddingSize = 2;
+  const transitionDuration = "0.3s";
+
+  return (
+    <Box
+      as="button"
```
Comment (johnmck95) [inline nutridine/src/components/navbar/MenuItem.js:11]: This is fine, but there are specific `Button` and `ButtonIcons` that can be used fyi

**Review by johnmck95 — COMMENTED** [review]
I'm wondering what this is trying to achieve exactly? I thought the MVP was to build the filtration page so users can search by macros?

**Comment by mehradadimi** [comment]
> I'm wondering what this is trying to achieve exactly? I thought the MVP was to build the filtration page so users can search by macros?

I just made this based on the Figma design. This is used to navigate between pages. From what I understood from the meeting, I thought we will have a Macro page and a Browse(meal) page

**Review by abraham-rostampoor — APPROVED** [review]
Looks great @mehradadimi! Great work!

**Comment by johnmck95** [comment]
Not that we have to do it fix this right now, but I noticed the nav bar is cutting off-screen content. Also wondering thinking nav bar doesn't need the entire screen to toggle between 3 pages when viewport is still very large (like 720px in the demo)

image

image

**Pull request merged**